doc/developer/workflow.rst

```diff
@@ -459,6 +443,24 @@ Guidelines for code review
   code change is large enough/significant enough to warrant such
   a requirement.
 
+For project members with merge permissions, the following patterns have
+emerged:
+
+- a PR with any negative reviews may not be merged.
```

Comment:
maybe clarify to mean "requested changes", since we can still merge with negative reviews present when a consensus has been reached

Reply:
updated.